### DISCUSSION TITLE
ALCOM cant find Unity Hub, and I cant either.

### DISCUSSION THREAD

**Discussion by Mello46514** [vrc-get/vrc-get#1719] [Q&A] [answered]
This is either me being dumb, or maybe something else. But **Im having trouble finding Unity Hub, and so is ALCOM.** It might be because im using the flatpak version of unity hub. and i know that packages from flatpak don't mix well with other packages. If Its an issue on specifically arch (which i doubt), let me know. 

On that note, im stumped, and i would like to know any solutions to this problem!

**also, if i am in the wrong place, Let me know immidatley!**

**Comment by anatawa12** [comment, marked as answer]
I think here is good place for this topic.

Flatpak unity hub problem is discussed at #1332 and adding flatpak unity hub to default search path is in #1586 but #1586 is blocked by upstream issue https://github.com/flathub/com.unity.UnityHub/issues/126 (has Pull Request but no one reviewed the PR yet) so please wait until upstream issue is resolved.

**Reply by Mello46514** [reply]
Alright, I'll just get the regular version of unity hub. Thanks for telling me!
